examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -119,27 +136,35 @@ def save_model_card(
 
 [Download]({repo_id}/tree/main) them in the Files & versions tab.
 
+"""
+    if "playgroundai" in args.pretrained_model_name_or_path:
+        model_description += """\n
+## License
+
+Please adhere to the licensing terms as described [here](https://huggingface.co/playgroundai/playground-v2.5-1024px-aesthetic/blob/main/LICENSE.md).
 """
     model_card = load_or_create_model_card(
         repo_id_or_path=repo_id,
         from_training=True,
-        license="openrail++",
+        license="openrail++" if "playgroundai" not in base_model else "other",
         base_model=base_model,
         prompt=instance_prompt,
         model_description=model_description,
         widget=widget_dict,
     )
     tags = [
         "text-to-image",
-        "stable-diffusion-xl",
-        "stable-diffusion-xl-diffusers",
         "text-to-image",
         "diffusers",
         "lora",
         "template:sd-lora",
     ]
-    model_card = populate_model_card(model_card, tags=tags)
+    if "playgroundai" in base_model:
+        tags.extend(["playground", "playground-diffusers"])
+    else:
+        tags.extend(["stable-diffusion-xl", "stable-diffusion-xl-diffusers"])
```

Comment:
Should these also be applicable, seeing that the playground model is essentially derivative?